composer.json

```diff
@@ -20,10 +20,7 @@
         "jakub-onderka/php-console-highlighter": "0.3.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0",
-        "symfony/finder": "~2.1",
-        "squizlabs/php_codesniffer": "~2.0",
-        "fabpot/php-cs-fixer": "~1.3"
```

Comment:
StyleCI makes it super easy for contributors to download a diff then can apply using "git apply" anyway. I think this is arguably faster than running php-cs-fixer on their system - it runs much slower for people with xdebug installed for example.